python/marvin/tools/cube.py

```diff
@@ -27,6 +27,7 @@
 from marvin.tools.quantities import DataCube, Spectrum
 from marvin.utils.datamodel.drp import datamodel
 from marvin.utils.general import FuzzyDict, get_nsa_data, gunzip
+import platform
```

Comment:
why is platform imported here?  It doesn't look like it's used.

Reply:
should be removed, please, @mstalbot